sdk/monitor/monitor-query/package.json

```diff
@@ -99,8 +99,10 @@
   "prettier": "@azure/eslint-plugin-azure-sdk/prettier.json",
   "dependencies": {
     "@azure/core-auth": "^1.3.0",
-    "@azure/core-http": "^2.0.0",
     "@azure/core-tracing": "1.0.0-preview.13",
+    "@azure/core-client": "^1.0.0",
+    "@azure/core-rest-pipeline": "^1.1.0",
+    "@azure/abort-controller": "^1.0.0",
```

Comment:
do you need it in the `src` directory? I see it in tests only but I could have missed it in `src`. If you're not using in `src`, please move it to `devDependencies` instead.

Reply:
moved to depdependencies